src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -155,7 +156,7 @@
     <Compile Include="System\AppContext\AppContext.Switch.Validation.cs" />
     <Compile Include="System\Collections\Generic\KeyNotFoundExceptionTests.cs" />
     <Compile Include="System\Collections\Generic\KeyValuePairTests.cs" />
-    <Compile Include="System\Collections\ObjectModel\CollectionTests.cs" />
+    
```

Comment:
Nit: unintentional extra line

Reply:
Thanks for pointing this out, it has been fixed locally and will be updated with the next code push